tests/multiparser_js_css/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -287,11 +289,11 @@ styled.span\`
 \`;
 
 styled.span\`
-  \${foo}: \${bar}
+  \${foo}: \${bar};
 \`;
 
 styled.span\`
-  \${foo}: \${bar}
+  \${foo}: \${bar};
```

Comment:
@evilebottnawi if we consider this case, previously we remove the semi also not right, exta semi is better, but

```js
const color= 'red'
const width = '1px'
const Button = styled.a`
  color: ${color}
  width: ${width}
`
```
remove this might cause problem, am I right?

Reply:
Why would anyone ever put a semicolon in `test` if they intend to use it as a property value? I could just as well see people reporting _not_ inserting the semicolon as a bug: “Why is `color: red` turned into `color: red;`, but not `color: ${test}` into `color: ${test};`?”

Reply:
@lydell so https://github.com/prettier/prettier/issues/2636 not solved, but it is in issues list (above), as I said earlier, I'm pretty sure there exists edge case when we break code, but it’s not so easy to solve, I understand, so maybe the solution is better than now

Reply:
@evilebottnawi [playground](https://deploy-preview-6794--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEcAeAHCAnGACSKAZ3wCEBXGGaAdWwEMMM5s8BePEgTwBs4ATAHQAjStSgADADpQ8eACTBh9InAC+MuYoAWEAG4sNshcAgZ6AR3LrNJ4TwhgA1ka3B6YGAEsDrk-y8iensBP0UISh4vKBtjRX4VbVDbRSJzMGiAcyMJAG4QABoQM29oImRQemxsCAB3AAUqhHKUej0IL35CkGEGZzgYAGV0rOQYbGsi7RgAWx4abS94NI84QealnyWuZHAicqLo1Vx6hkyZ+mQAM3oeVSKAKyI0Uj6nAcH6GbgAGWi4a63e4gJ5oQZZPgARXIEHggLucCK5mwx12ymEcB43Qw2GiMBonRg2mQAA4AAxImqqOiMXY4uDHAzdKywuCnMwtEAqAC0MQEAm62DgVi8QtO9HOlyQNwRRVUMy8YwmiJARAhcGhrPhwJgwQJ-CJyAATEVxvQvFEoJkAMIQGYXXZQaAAorkVQAFWCLRlwL01gAklB+AghmBcRgYABBIODGC8AHSoHqNRAA) it's solved, haha, magic~~

Reply:
different case, if you read #6790  carefule , you can understand why they are different,

add a prop before the last one, you'll see [link](https://deploy-preview-6794--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEcAeAHCAnGACSKAZ3wCEBXGGaAdWwEMMM5s8BePEgTwBs4ATAHQAjStSgADADpQ8eACTBh9InAC+MuYoAWEAG4sNshcAgZ6AR3LrNJ4TwhgA1ka3B6YGAEsDrk-y8iensBP0UISh4vKBtjRX4VbVDbADMICCQTInMwaIBzIwkAbhAAGhAzb2giZFB6bGwIAHcABXqEGpR6PQgvfjKQYQZnOBgAZRz85Bhsa3LtGABbHhptL3hsjzgxjvWfda5kcCIa8ujVXBaGPMX6ZBT6HlVygCsiNFJhp1Gx+kW4AAy0Tg90ezxAbzQY3yfAAiuQIPBQU84OVzNgLkdlMI4DwBhhsNEYDQ+jBtMgABwABjRjVUdEYRwJcAuBgGVkRcCuZk6IBUAFoYgIBANsHArF4xVd6Dc7kgHijyqpFl5prNUSAiDC4PDOcjwTBgiT+GTkAAmcozeheKJQPIAYQgi1uRyg0BB5XIqgAKsFOgrwXprABJKD8BDjMCEjAwACCYbGMF4IPlYPUaiAA) and [link2](https://deploy-preview-6794--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEcAeAHCAnGACSKAZ3wCEBXGGaAdWwEMMM5s8BePEgTwBs4ATAHQAjStSgADADpQ8eACTBh9InAC+igBYQAbizUy5iiBnoBHcuoVKeEMAGs1AbkPX6YGAEs9B2df6eRPTCfPy+RsAQlDyeUFYA9PF4UBCccAC2nq6K-CqaAs54iWmZrgBmEBBI1kSmYLEA5mpFSTr0PJYyEk4gADQgJl7QRMig9NjYEADuAArjCCMo9DoQnvx9IMIMDnAwAMp1jcgw2Jb9mjDpPDSanvC17nB7C3fed1zI4EQj-bGquDMGA10vRkGV2qp+gArIhoUjbey7Pb0dJwAAysTgYIhcGhsL2jT4AEVyBB4NieJCQKZsP9PsphHAeBsMNhYjAaGsYJpkAAOAAM-VZEFUdEYn1ZcH+eg2FjJcEBJkWIBUAFo4gIBBtsHALJ4dYD6MDQUhwZTcSBVJljqcLURCXASfKKVSYMFOfxucgAEz9E70TwxKANADCEHSIM+KTiG3IqgAKsFFmaqTpLABJKD8BD7MBsjAwACCWb2MF4WNNOLUaiAA)